grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -1152,69 +1152,90 @@ class StockMovementService {
             }
             if (pickPageItem) {
                 AvailableItem availableItem = pickPageItem.getAvailableItem(data.binLocation, data.lotNumber)
+                ApplicationTagLib g = grailsApplication.mainContext.getBean(ApplicationTagLib.class)
 
                 if (!availableItem) {
-                    String lotNumberErrorMessage = data.lotNumber ?: "empty"
-                    String binLocationErrorMessage = data.binLocation ?: "empty"
+
+                    String lotNumberName = data.lotNumber ?: g.message(code: "default.label", default: "Default")
```

Comment:
I used Constants.DEFAULT_LOT_NUMBER and Constants.DEFAULT_BIN_LOCATION, but this is much better. 

However, let's do this to be consistent.
```
String lotNumberName = data.lotNumber ?: g.message(code: "default.noLotNumber.label", default: Constants.DEFAULT_LOT_NUMBER)
```

Reply:
```
String binLocationName = data.binLocation ?: g.message(code: "default.noBinLocation.label", default: Constants.DEFAULT_BIN_LOCATION)
```